CoreScriptsRoot/Modules/BackpackScript3D.lua

```diff
@@ -157,11 +163,11 @@ local function OnHotbarEquipPrimary(actionName, state, obj)
 	for tool, slot in pairs(Tools) do
 		if slot.hovered then
 			slot.OnClick()
+			return
```

Comment:
Can this be moved over to the `GuiService.SelectedCoreObject` style as well? Or will it not work for this script?